src/app/shared/components/Field/PlayerCard/index.tsx

```diff
@@ -32,9 +34,9 @@ const PlayerCard = (props: PlayerCardProps) => {
   const isPlayerSelected = !!player?.id
 
   return (
-    <div className={`${className} h-[30px]`}>
+    <div className={`${className}`}>
       <div
-        className="box-border border-black rounded cursor-pointer flex flex-col p-1 min-h-[50px] min-w-[80px] w-fit border hover:border-cyan-300 relative"
+        className="box-border border-black rounded cursor-pointer flex flex-col min-h-[50px] w-[80px] w-fit border p-1 hover:border-cyan-300 relative"
```

Comment:
are we intending on keeping the black border in the finished product?

Reply:
Hmm I'm not sure about this yet? I had a look without borders, I kinda like them as clear delineators but needs more thought